samples/smoketest/key_vault_secrets_async.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
Dunno whether we need license headers on these files, but it would be:
```py
# ------------------------------------
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.
# ------------------------------------
```

Reply:
@danieljurek this is an interesting question, should I add this header to all my smoke-test files?